packages/tdesign-vue-next/site/docs/getting-started.en-US.md

```diff
@@ -145,6 +145,8 @@ module.exports = {
 In Nuxt 3, you can install the Nuxt module @tdesign-vue-next/nuxt
```

Comment:
```suggestion
In Nuxt 3, you can install the Nuxt module @tdesign-vue-next/nuxt
```
In Nuxt 3, you can install the Nuxt module [@tdesign-vue-next/nuxt](https://nuxt.com/modules/tdesign-vue-next)